Crud-3-2-2022/controller/loginController.php

```diff
@@ -0,0 +1,66 @@
+<?php
+require "./config/myConfig.php";
+class LoginController extends Database
+{
+
+   public function __construct()
+   {
+      parent::__construct();
+      $this->login();
+   }
+   public function requiredFeild(){
+      if(empty($_POST['username']) || empty($_POST['password'])){
+         return true;
+      }else{
+         return false;
+      }
+   }
```

Comment:
dùng toán tử tối ưu if elst

Return (empty($_POST['username']) || empty($_POST['password']));